harness/determined/pytorch/_pytorch_context.py

```diff
@@ -116,6 +151,12 @@ def get_per_slot_batch_size(self) -> int:
         """
         return self._per_slot_batch_size
 
+    def get_experiment_config(self) -> Dict[str, Any]:
+        return self.exp_conf
```

Comment:
What about the experiment config needs to be given to the end user?  I would very much like to avoid exposing the full experiment config to the end user because the experiment config is not a stable api for reading from, it's only stable for writing to (we might shim it at runtime without your code being aware of the new structure).

Reply:
well, some code in our examples is using it. i agree but this probably requires some more discussion because it would be a breaking change to just get rid of it.